weather-app/src/app/modules/app/components/current-weather/current-weather.component.html

```diff
@@ -0,0 +1,104 @@
+<!-- <div class="container current-weather w-50 m-3 p-2">
```

Comment:
remove this commented code and check if there is scss classes which will become unused after this